GitUI/CommandsDialogs/FormCommit.cs

```diff
@@ -186,6 +187,7 @@ public FormCommit([NotNull] GitUICommands commands, CommitKind commitKind = Comm
             : base(commands)
         {
             ThreadHelper.ThrowIfNotOnUIThread();
+            _joinableTasks = new JoinableTaskCollection(ThreadHelper.JoinableTaskContext);
```

Comment:
💭 Not sure if there's a difference between this and `ThreadHelper.JoinableTaskContext.CreateCollection()`

Reply:
There isn't -- today. But we recommend the `CreateCollection` method when you don't care about the concrete type you're creating so that our "factory" can create a collection best suited for the VS app (if it were to change in the future). So ya, using `ThreadHelper.JoinableTaskContext.CreateCollection()` is preferred.

---
In reply to: [258625981](https://github.com/gitextensions/gitextensions/pull/6273#discussion_r258625981) [](ancestors = 258625981)